src/display/api.js

```diff
@@ -613,7 +615,7 @@ class PDFDataRangeTransport {
 }
 
 /**
- * Proxy to a PDFDocument in the worker thread. Also, contains commonly used
+ * Proxy to a `PDFDocument` in the worker thread that contains commonly used
  * properties that can be read synchronously.
```

Comment:
This comment is probably *very* old, and with less than a handful of exceptions almost all of it is now *a*synchronous. Hence I wonder if it the second sentence, or its contents, even makes sense here still!?

Also, compare with the `PDFPageProxy` class JSDocs below.